platform/security/sts/security-sts-ldapclaimshandler/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -58,6 +58,7 @@
             <property name="memberNameAttribute" value="member"/>
             <property name="groupBaseDn" value="ou=groups,dc=example,dc=com"/>
             <property name="propertyFileLocation" value="${ddf.home}/etc/ws-security/attributeMap.properties"/>
+            <property name="overrideCertDn" value="false" />
```

Comment:
Don't you need to update the `ClaimsHandlerManager` so when it registers the two handler types it sets this new property?